src/ensembl/production/metadata/grpc/adaptors/genome.py

```diff
@@ -647,3 +650,37 @@ def fetch_organisms_group_counts(self, species_taxonomy_id=None, release_version
 		with self.metadata_db.session_scope() as session:
 			# TODO check if we should return a dictionary instead
 			return session.execute(query).all()
+
+	def fetch_related_assemblies_count(self, species_taxonomy_id=None, release_version=None):
+		o_species = aliased(Organism)
+		o = aliased(Organism)
+		if not release_version:
+			# Get latest released organisms
+			query = db.select(
```

Comment:
Why not just counting and return the number directly here?

Reply:
That's a good idea, thank you for your suggestion

Reply:
Done